src/plugins/apimon/apimon.cpp

```diff
@@ -333,9 +334,10 @@ apimon::apimon(drakvuf_t drakvuf, const apimon_config* c, output_format_t output
     {
         drakvuf_load_dll_hook_config(drakvuf, c->dll_hooks_list, c->print_no_addr, &this->wanted_hooks);
     }
-    catch (int e)
+    catch (UserHookException& exc)
     {
-        fprintf(stderr, "Malformed DLL hook configuration for APIMON plugin\n");
+        std::cerr << "Loading DLL hook configuration for APIMON plugin failed" << std::endl
```

Comment:
```suggestion
        std::cerr << "Loading DLL hook configuration for APIMON plugin failed\n"
```
`std::endl` isn't a simple replacement for `\n`, it also performs a flush and we don't need it here :)